### PR TITLE
Update for Elixir 1.5.1

### PR DESCRIPTION
**Pull request by ijcd**

### CODE REVIEW

**Comment by copenhas** [comment]
Woah out of nowhere. Thanks for the PR.

File: mix.exs [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
+      deps: deps(),
+
+      # Docs
       name: "ropex",
-      source_url: source_url,
-      deps: deps,
-      docs: [
-        source_url_pattern: "#{source_url}/blob/#{version}/%{path}#L%{line}"
-      ]
+      source_url: "https://github.com/ijcd/ropex",
```
Comment (copenhas) [inline mix.exs]: Since this is being contributed back to my fork, should this be `copenhas/ropex`?

**Comment by ijcd** [comment]
So I just did a presentation at ElixirConf where I talked about how I was
using this (building an editor). If you want to maintain it, sure. I was
hoping it would get published to hex.pm and updated etc. I plan to use it a
bit so would be happy to take over maintenance if it's not actively
maintained anymore. I didn't just want to gank it since you wrote it
though. What's your preference?

--Ian

**Comment by copenhas** [comment]
Very cool, I got sidetracked watching your presentation actually. 

Pretty obvious it's not actively maintained. I appreciate the offer but if you are building on top of it, I'll probably slow you down.

**Comment by ijcd** [comment]
Very well. Thanks for writing it!

**Pull request merged**